package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tree-sitter-json",
-  "version": "0.14.0",
+  "version": "0.14.1",
```

Comment:
I think the version bump is necessary? And I think we don’t want it to be a major version bump? Not sure.

Reply:
Maybe it should be major because we renamed the entry rule to `document`?

Reply:
I think let's do `0.15.0`. Currently I'm not using the package version number to communicate grammar changes, but I do try to have the minor version track the `tree-sitter` minor version, in order to help communicate ABI compatibility.